ext/node/polyfills/process.ts

```diff
@@ -54,41 +54,14 @@ export let platform = "";
 // TODO(kt3k): This should be set at start up time
 export let pid = 0;
 
-// We want streams to be as lazy as possible, but we cannot export a getter in a module. To
-// work around this we make these proxies that eagerly instantiate the underlying object on
-// first access of any property/method.
-function makeLazyStream<T>(objectFactory: () => T): T {
-  return new Proxy({}, {
-    get: function (_, prop, receiver) {
-      // deno-lint-ignore no-explicit-any
-      return Reflect.get(objectFactory() as any, prop, receiver);
```

Comment:
cc @bartlomieju @mmastrac 

This breaks code in `_streams.mjs` because of the use of Proxy:

```
divy@mini /tmp> ~/gh/deno/target/debug/deno run -A npm:nuxi@latest init my-app

[5:25:30 PM]  ERROR  'get' on proxy: property '_eventsCount' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '2' but got '1')

  at _addListener (ext:deno_node/_stream.mjs:1884:9)
  at Proxy.addListener (ext:deno_node/_stream.mjs:1913:14)
  at Proxy.Readable.on (ext:deno_node/_stream.mjs:3275:39)
  at Proxy.Readable.pipe (ext:deno_node/_stream.mjs:3177:11)
```

when its trying to update the eventsCount. I reverted it back to not lazy load the stdin/out/err streams which fixed the issue.